src/telemetryGenerator.node.ts

```diff
@@ -776,6 +776,11 @@ function generateDocumentation(fileNames: string[]): string | undefined {
                             const properties = comptePropertyGroupsForIntersectionTypes(type, typeChecker);
                             const comment = getCommentForUnions(type);
                             groups.push({ description: comment, properties });
+                        } else if (
+                            type.kind === ts.SyntaxKind.UndefinedKeyword ||
+                            type.kind === ts.SyntaxKind.NeverKeyword
```

Comment:
Minor tweek to the telemetry generator, we were generating warnings about this in subsequent line when we didn't have to.